src/main/java/uk/gov/hmcts/reform/civil/model/citizenui/CcdDashboardDefendantClaimMatcher.java

```diff
@@ -294,4 +294,14 @@ public boolean isClaimantDefaultJudgement() {
             && caseData.getPaymentTypeSelection() != null;
     }
 
+    @Override
+    public boolean isPartialAdmissionAccepted() {
+        return false;
```

Comment:
does it suppose to be hardcoded?

Reply:
there is no logic defined yet, it is auto generated